README.md

```diff
@@ -1,29 +1,47 @@
-serocalculator package
-=====================
-
-------------------------------------------------------------------------
-
-<!-- badges: start -->
-[![R-CMD-check](https://github.com/UCD-SERG/serocalculator/workflows/R-CMD-check/badge.svg)](https://github.com/UCD-SERG/serocalculator/actions)
-<!-- badges: end -->
-
-Antibody levels measured in a (cross–sectional) population sample can be
-translated into an estimate of the frequency with which seroconversions
-(infections) occur in the sampled population. Formulated simply: the
-presence of many high titres indicates that many subjects likely
-experienced infection recently, while low titres indicate a low
-frequency of infections in the sampled population.
-
-The serocalculator script was designed to use the longitudinal
-response characteristics by means of a set of parameters characterizing
-the longitudinal response of the selected serum antibodies.
-
-## Installation
-
-You can install the development version from
-[GitHub](https://github.com/) with:
-
-``` r
-install.packages("devtools")
-devtools::install_github("ucd-serg/serocalculator")
-```
+serocalculator package
+=====================
+
+------------------------------------------------------------------------
+
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/UCD-SERG/serocalculator/workflows/R-CMD-check/badge.svg)](https://github.com/UCD-SERG/serocalculator/actions)
+<!-- badges: end -->
+
+Antibody levels measured in a (cross–sectional) population sample can be
+translated into an estimate of the frequency with which seroconversions
+(infections) occur in the sampled population. Formulated simply: the
+presence of many high titres indicates that many subjects likely
+experienced infection recently, while low titres indicate a low
+frequency of infections in the sampled population.
+
+The serocalculator script was designed to use the longitudinal
+response characteristics by means of a set of parameters characterizing
+the longitudinal response of the selected serum antibodies.
+
+## Installation
+
+You can install the development version from
+[GitHub](https://github.com/) with:
+
+``` r
+install.packages("devtools")
+devtools::install_github("ucd-serg/serocalculator")
+```
+
+**A Note for Windows Users**
+
+Windows users will need to install Rtools, which contains a collection of tools for building and employing R packages that are still in development. 

```

Comment:
Did the version of Rtools installed by devtools not work? I can't remember what happened with that.